### DISCUSSION TITLE
Is there more master workspace rules?

### DISCUSSION THREAD

**Discussion by rossvold** [hyprwm/Hyprland#7642] [Q&A]
https://wiki.hyprland.org/Configuring/Master-Layout/#workspace-rules

Does this mean, that the only workspace rule that can be set here is the orientation?

workspace = 6, layoutopt:orientation:right -- Works as documentation says
workspace = 6, layoutopt:mfact:0.75 -- Does not work, (as i expected)

Is there any config that allows me to set a workspace mfact? Or do i have to use a dispatcher somehow?

I currently solve this with a keybind
bind = $mainMod, P, layoutmsg, mfact exact 0.75
which works perfectly fine, but i would prefer it to be a rule for a spesific workspace instead of it being a keybind.

I also looked here:
https://wiki.hyprland.org/Configuring/Workspace-Rules/
but couldn't find anything related.

**Comment by robclancy** [comment]
Oh I have been trying to get per workspace mfact to work for a while now, I never realized it just doesn't work but will have to use your bind I guess.

**Reply by rossvold** [reply]
Yeah you prob just followed the exact same path as me, using a dispatcher like i suggested does work.
